azure/templates/pds-tests.yml

```diff
@@ -21,7 +21,6 @@ steps:
       - /ptl/azure-devops/personal-demographics/TEST_PATIENT_ID
       - /ptl/azure-devops/personal-demographics/AUTH_TOKEN_EXPIRY_MS
       - /ptl/azure-devops/personal-demographics/AUTH_TOKEN_EXPIRY_MS_INT
-      - /ptl/azure-devops/personal-demographics/SPINE_HOST
```

Comment:
Removed the only use for this env, so have removed it's declaration in the pipeline build.